tikv-client/src/main/java/com/pingcap/tikv/key/CommonHandle.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2020 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pingcap.tikv.key;
+
+import com.pingcap.tikv.codec.Codec;
+import com.pingcap.tikv.codec.CodecDataInput;
+import com.pingcap.tikv.exception.CodecException;
+import com.pingcap.tikv.util.FastByteComparisons;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CommonHandle implements Handle {
+  private final byte[] encoded;
+  private final int[] colEndOffsets;
+
+  public CommonHandle(byte[] encoded) {
```

Comment:
TiSpark provides `newCommonHandle(DataType[] dataTypes, Object[] data)` to get CommonHandle from dataTypes and data.
Do we need it too?